### PR TITLE
CLI: remove step logs

### PR DESCRIPTION
**Pull request by zc-devs**
On top of #3451, addresses [PR note](https://github.com/woodpecker-ci/woodpecker/pull/3451#discussion_r1505438144)

Not tested.

### CODE REVIEW

File: woodpecker-go/woodpecker/interface.go [hunk 1]
```diff
@@ -116,7 +116,7 @@ type Client interface {
 	Deploy(repoID, pipeline int64, env string, params map[string]string) (*Pipeline, error)
 
 	// LogsPurge purges the pipeline logs for the specified pipeline.
-	LogsPurge(repoID, pipeline int64) error
+	LogsPurge(repoID, pipeline int64, step int64) error
```
Comment (6543) [inline woodpecker-go/woodpecker/interface.go:119]: is breaking ...

you could make step optional via:
```suggestion
	LogsPurge(repoID, pipeline int64, step ...int64) error
```

Comment (zc-devs) [reply woodpecker-go/woodpecker/interface.go]: Thanks, that is what I meant.

**Comment by 6543** [comment]
Well you still have to adjust the code as `step` is now not int64 but of type []int64 ... 

So instead of:
```
if step != 0 {
...
````

use
```
if len(step) != 0 {
  stepID := step[0]
  ...
````

**Comment by zc-devs** [comment]
TLDR: Made separate library call. Should not be a breaking too.

---

```
LogsPurge(repoID, pipeline int64, step ...int64) error
```
Looks like it is not many step IDs, but any optional argument. So, I could make it
```
LogsPurge(repoID, pipeline int64, optArg ...int64) error
```
Next question is why are optional args integers only? Then it should be
```
LogsPurge(repoID, pipeline int64, optArg ...Interface{}) error
```
Then we should distinct that arguments somehow. So, it probably would be a map (argName:argValue).
Looks over complicated to me.

**Comment by zc-devs** [comment]
Let's try. Add `build` label, please.
